### DISCUSSION TITLE
Add "Merge conflicts" flag on PR when looking at the list of the PR

### DISCUSSION THREAD

**Discussion by VincentLanglet** [community/community#42916] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

When looking at the list of Pull request, this is what we can see 
![image](https://user-images.githubusercontent.com/9052536/210057980-9e16fd85-ff15-4a83-acf3-026c52e280ec.png)

- We can easily see if the Ci is green ✅ 
- We can easily see if there is request changes ✅ 
- We can't see if the PR has merge conflict 🔴 

I personally would love a notification when my PR have conflict, but this would be a next-lever feature.
Until that, it would be great that Github add a `Merge conflict` information in this list.

**Comment by jfullstackdev** [comment]
if I'm getting that correctly, you can add the label on your own, isn't it ?

**Reply by VincentLanglet** [reply]
> if I'm getting that correctly, you can add the label on your own, isn't it ?

You don't get the goal.
If I go to the PR and see the conflict, I'll fix it instead of adding a label.
But I don't want to check every opened PR, every hour to know if there is a merge conflict to solve.

What I would like is an easy way to check if among all these PR, there are one with merge conflicts I need to solve, or if I can just go back to work.

I was the same for the CI checks or the change requested checks. We could check by ourselves every PR and add label on PR with failures, but it's so much easy to see with this view that none of these PR have CI failure without any actions from me.

**Comment by mvandermade** [comment]
This would be very helpful indeed !
